lstchain/high_level/interpolate.py

```diff
@@ -361,8 +362,10 @@ def interpolate_gh_cuts(
     ----------
     .. [1] https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.griddata.html
     """
-    interp = GridDataInterpolator(grid_points=grid_points, params=gh_cuts)
-    gh_cuts_interp = interp(target_point, method=method)
+    interp = GridDataInterpolator(
+        grid_points=grid_points, params=gh_cuts, method=method
+    )
+    gh_cuts_interp = interp(target_point)
```

Comment:
If extrapolation is needed I'd suggest to create a Estimator object (same for the AL_CUTS), just like it is done for RAD_MAX tables in pyirf. If you don't want extrapolation this should be fine.